mxcube3/routes/SampleCentring.py

```diff
@@ -128,6 +128,16 @@ def state_cb(state, motor=motor, **kw):
         except Exception as ex:
             logging.getLogger('HWR').exception(str(ex))
 
+    def zoom_state_cb(state, motor='zoom', **kw):
+        movable = Utils.get_movable_state_and_position('zoom')
+        signals.motor_state_callback(movable['zoom'], **kw)
+
+    try:
```

Comment:
Why are you adding these back again, I think we removed them in a previous PR ?

Reply:
There is a comment in Utils that says this:
   ```
   def get_centring_motors_info():
       # the centring motors are: ["phi", "focus", "phiz", "phiy",
       # "zoom", "sampx", "sampy", "kappa", "kappa_phi"]
   ```

So zoom is already handled in the code above

Reply:
I will double check tomorrow in the beamline

Reply:
You are right.